Podcast/UIImage+.swift

```diff
@@ -20,4 +20,12 @@ extension UIImage {
         UIGraphicsEndImageContext()
         return newImage!
     }
+
+    func withAlpha(_ alpha: CGFloat) -> UIImage? {
+        UIGraphicsBeginImageContextWithOptions(size, false, scale)
+        draw(at: .zero, blendMode: .normal, alpha: alpha)
+        let newImage = UIGraphicsGetImageFromCurrentImageContext()
+        UIGraphicsEndImageContext()
```

Comment:
This doesn't have to do with this PR but before you merge in can you make the last updated label on a series feed cell to be two lines

Reply:
<img width="254" alt="screen shot 2018-03-05 at 11 04 29 pm" src="https://user-images.githubusercontent.com/14966713/37013820-4c72fbfc-20ca-11e8-958e-19a82aa21ba4.png">
We have this issue rn